### ISSUE TITLE
Can Aurora be applied to urban scale?

### ISSUE THREAD

**Issue by DoorMaXX** [microsoft/aurora#101]
Hello, thank you for this meaningful research！
I had noticed that Aurora can accommodate weather data at different resolutions, and apply to regional scale through finetuning. If I use Aurora to conduct urban microclimate forecast(simulation grid is 10-100m), does the huge resolution and parameter settings gap between finetuning dataset I provided and Aurora pretained dataset lead to a loss of feasibility?

**Comment by wesselb** [comment]
Hey @DoorMaXX! Thanks for your kind words and for opening an issue. :)

On a practical level, I think Aurora can be fine-tuned to do urban microclimate forecasting. The huge resolution might pose issues, depending on how large the data really is, but this can be mitigated by adjusting the patch size.

However, how well pretraining would help Aurora to do urban microclimate forecasting is a big unknown. I really have no idea! It could help, since it's also weather forecasting; but it could also only help to a limited extent, since urban microclimates could demonstrate different dynamics. I'm sorry for the inconclusive answer, but I think that the only way to really find out is to try!

**Comment by DoorMaXX** [comment]
@wesselb A detailed and valuable answer, thanks!
